integration/image/inspect_test.go

```diff
@@ -19,9 +19,9 @@ func TestImageInspectEmptyTagsAndDigests(t *testing.T) {
 	client := testEnv.APIClient()
 	ctx := context.Background()
 
-	danglingId := environment.GetTestDanglingImageId(testEnv)
+	danglingID := environment.GetTestDanglingImageId(testEnv)
```

Comment:
Was this meant to go in the first commit? 🙈 

```diff
	danglingID := environment.GetTestDanglingImageID(testEnv)
```

Reply:
🙈  Fixing it up!

Reply:
Done, I mistakenly did a `--fixup` of both tests to the same commit 😅